File: Sb.java
import java.util.*;
import java.lang.*;
import java.io.*;

public class Main{
	public static void main (String[] args){
        Scanner sc= new Scanner(System.in);     
System.out.print("Enter your year of birth (YYYY): ");  
int year = sc.nextInt();  
System.out.print("Enter your current salary: "); 

 while (!sc.hasNextDouble()) {
      System.out.println("Input is not a number. Please enter only digits.");
      sc.nextDouble();
    }
  
double salary = sc.nextDouble();  

if(year>2000 && year<2004){
salary = ( 0.05 * salary ) + salary;
}
else if(year>1990 && year<2001){
salary =  ( 0.10 * salary) + salary;
}
else if (year>1970 && year<1991){
salary =  ( 0.20 * salary) + salary;
}
else{
salary = salary;
}

System.out.println("Your salary and bonus total is " + salary); 

sc.close();
	}
}

File: Sb.js
<!DOCTYPE html>
<html>
<body>

<h2>Bonus & Salary Calculator</h2>
<h4>Task 1</h4>

<p id="demo"></p>

<script>
let year = parseInt(prompt("What is your year of birth (YYYY): "));
let salary = parseInt(prompt("What is your current salary (please enter numbers only): "));

if(year>2000 && year<2004){
    salary = ((5/100)*salary)+salary;
    document.write("Your expected bonus and salary is "+ salary);
}
else if(year>1990 && year<2001){
    salary = ((10/100)*salary)+salary;
    document.write("Your expected bonus and salary is "+ salary);
}
else if (year>1970 && year<1991){
    salary = ((20/100)*salary)+salary;
    document.write("Your expected bonus and salary is "+ salary);
}
else{
    document.write("Unfortunately, you do not qualify for the bonus");
}

</script>

</body>
</html>
